### PR TITLE
Add eth-gas-reporter for unit tests

### PR DESCRIPTION
**Pull request by patitonar**
Add [eth-gas-reporter](https://github.com/cgewecke/eth-gas-reporter) for gas usage reports on unit tests.

Related to #47

### CODE REVIEW

**Comment by patitonar** [comment]
@akolotov unit test were using 3 validators and 2 required signatures for `submitSignature`, `executeAffirmation` and `executeSignatures`. I added new tests with 5 validators and 3 required signatures. Also I added gas usage report on README file under `test` section. Other alternatives are to move the report to a separate file y linked it from README file or put the report on its own section.

File: README.md [hunk 1]
```diff
@@ -62,6 +62,127 @@ Please the [README.md](deploy/README.md) in the `deploy` folder for instructions
 npm test
 ```
 
+##### Gas usage report
```
Comment (akolotov) [inline README.md]: This information is hard to analyse especially if you are not familiar with the bridge contracts code execution flow. Please take a look at the review comment to consider more practical example of the report that it make sense to add to the README.

**Comment by patitonar** [comment]
@akolotov I added a summarized gas consumption section for `Native-to-ERC` mode, if you think it is OK I can move forward with the other modes. Also since it is a lot of information, I think it will be best to move this section to a different file and add a link to it on the `README`

**Comment by akolotov** [comment]
>  I think it will be best to move this section to a different file and add a link to it on the README
Right. Let's create a separate file. 
I am OK with `Native-to-ERC` section. Good job!

**Pull request merged**